arkane/molpro.py

```diff
@@ -302,44 +306,45 @@ def loadEnergy(self, frequencyScaleFactor=1.):
                                  ' MRCI, MRCI+Davidson are supported')
             # Search for E0
             for line in lines:
-                if f12a:
+                if f12 and f12a:
                     if 'CCSD(T)-F12a' in line and 'energy' in line:
-                        E0 = float(line.split()[-1])
+                        e0 = float(line.split()[-1])
                         break
-                    if 'Electronic Energy at 0' in line:
-                        E0 = float(line.split()[-2])
-                        break
-                elif f12b:
+                elif f12 and f12b:
                     if 'CCSD(T)-F12b' in line and 'energy' in line:
-                        E0 = float(line.split()[-1])
-                        break
-                    if 'Electronic Energy at 0' in line:
-                        E0 = float(line.split()[-2])
+                        e0 = float(line.split()[-1])
                         break
                 elif mrci:
                     # First search for MRCI+Davidson energy
                     if '(Davidson, relaxed reference)' in line:
-                        E0 = float(line.split()[3])
+                        e0 = float(line.split()[3])
                         logging.debug('Found MRCI+Davidson energy in molpro log file {0}, using this value'.format(
                             self.path))
                         break
-            if E0 is None and mrci:
+                elif not f12:
+                    if 'Electronic Energy at 0' in line:
```

Comment:
Just curious, do you know when (if ever) this gets used?

Reply:
Not really. This is actually the original line we were parsing so far for non-F12 sp jobs. The on;y thing "different" about the test I added is that it is QZ (`basis=aug-cc-pvqz`). Maybe that changes something in the output format.